pkg/apis/v1beta1/labels.go

```diff
@@ -51,6 +51,10 @@ func init() {
 	)
 }
 
+const (
+	TerminationFinalizer = Group + "/termination"
```

Comment:
Had we mistakenly removed this?

Reply:
No, we weren't using a termination finalizer in the `karpenter.k8s.aws` group up to this point